include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1028,10 +1028,12 @@ class FEPointEvaluation
    * batch.
    */
   void
-  compute_evaluate_fast(const unsigned int                  n_shapes,
-                        const unsigned int                  qb,
-                        vectorized_value_type &             value,
-                        interface_vectorized_gradient_type &gradient);
+  compute_evaluate_fast(
+    const EvaluationFlags::EvaluationFlags &evaluation_flags,
```

Comment:
```suggestion
    const EvaluationFlags::EvaluationFlags evaluation_flags,
```

Reply:
This is something we use in the whole file for every function that needs `EvaluationFlags`. If we want to change this we should do this in a separate PR.